### ISSUE TITLE
VoiceOver: Text input blocks no longer recognized as editing fields

### ISSUE THREAD

**Issue by twstokes** [wordpress-mobile/WordPress-iOS#21399]
A [user report from Elena B.](https://wordpress.slack.com/archives/C02RP4X03/p1692747057094999) in .org Slack:

> Since 23.0 of iOS/iPadOS JetPack app released August 21st, the editor does not work as expected with VoiceOver; in few words, the fields in post creation are no longer recognized as editing fields. Post title, paragraph blocks, HTML code editor...
Consequences: no possibility for Input Braille Screen; no way to select, cut, paste, copy text or move insertion point (cursor) to desired position. Consequently no ability to paste from a MarkDown editor to JetPack's editor.
Bluetooth keyboard? Yeah, tried, but while editing works with standard non-voiceover select commands, when I press Enter key to add a new block, focus jumps backwards in a random block making me lose the point. Instead of quickening my job, hardware keyboard ends up slowing it down.

| 22.9 | 23.0 |
| - | - |
| <video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/2092798/b9d71f81-188f-480c-b112-69d420f3b7c3" alt="Screen recording of Braille input working correctly in 22.9" /> | <video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/2092798/a4860e23-5406-48bb-b578-d15d20786c47" alt="Screen recording of Braille input not working correctly in 23.0" /> |

**Comment by elettrona** [comment]
This concerns JetPack (and I assume also WordPress) mobile app version 23.0. I am not talking about web browser, currently.
I am the person who found and encountered the issue, wrote on Slack and @twstokes reported it here.
I attempted to create a new video with the real issue as the original author might have recorded it from Safari
Expected behaviour: when focusing the post title, the keyboard would be shown while VoiceOver says "text field. Is editing" - as a standard edit field. Same on a single block from the block editor.
Until 22.x versions it worked, on 23.0 it has been broken.
Current behaviour: VoiceOver continues saying "text field. Double tap to editing" behaving like the field is closed, and it isn't.
This affects fast writing mode -input braille screen- and also normal editing functions using VoiceOver.
Pure HTML code view is also affected.
And, as it isn't recognized as a standard edit field, when you use a Bluetooth keyboard and press Enter key to create a new block, focus randomly jumps somewhere else, slowing everything down.
I attempt to place a video link, as it's too big in size for being uploaded here.
Hopefully you get it
Elena Brescacin
https://e.pcloud.link/publink/show?code=XZSyR2ZTGm0XeSvhPR59OPzkxMLjuYO5OJX

**Comment by twstokes** [comment]
Thank you very much @talksina! We really appreciate the video. We've confirmed the problem with 23.0 and are working on a fix. We'll loop back to this issue with any updates.

**Comment by elettrona** [comment]
yeah, thanks; I'll wait for further updates, meanwhile both plugins -JetPack and Gutenberg- received an automatic update in the WordPress website. Clearly from the app nothing has changed, I'll wait! 
I take the chance to add another note: accessibility labels of post title, start writing, and generally blocks interface, are untranslated -all in English- while in editing mode. I'm not native English and use Italian interface. 
While if the editor's keyboard is closed, they seem translated. 
I have attached this note here, as I suspect it's part of same issue affecting VoiceOver's behaviour. Be patient, I'm not a coder, just a tester. Thanks for attention

**Comment by twstokes** [comment]
> accessibility labels of post title, start writing, and generally blocks interface, are untranslated -all in English- while in editing mode. I'm not native English and use Italian interface.

Thanks once again for the great feedback @talksina! We're looking into this as well.

**Comment by elettrona** [comment]
Update: the 23.0.1 release was pushed on the app store; keyboard problem has been solved, but there are still some serious issues on focus/cursor management and labels translation. I'm going to open another one ASAP.

**Comment by geriux** [comment]
Hey there @talksina!

Glad to know that the keyboard problem is now solved.

Regarding the missing translations, we'll be addressing that issue in the next app update, thank you again for reporting that bug.

>  but there are still some serious issues on focus/cursor management 

We are sorry you are still having issues with the editor, please let us know either in this ticket or in a new one the current problems so we can address them as well.

Thanks again for your feedback!

**Comment by tiagomar** [comment]
Closing as it was fixed in https://github.com/WordPress/gutenberg/pull/53895.
